bin/cleanSecurityserviceTest.sh

```diff
@@ -1,12 +1,7 @@
 #!/bin/bash
 
-# x86_64 or arm64 for make run option
-[ "$(uname -m)" != "x86_64" ] && USE_ARM64="arm64"
-
-MAKEFILE=$(dirname "$0")/../Makefile
-
 echo "Info: Clean Securityservice's test data."
 
-make -f ${MAKEFILE} del-token ${USE_ARM64} jerry
+./bin/setupSecurityAccount.sh -userdel
```

Comment:
The setupSecurityAccount.sh is under bin/security foloder.  Same error in bin/setupSecurityserviceTest.sh.

Reply:
Thanks! Core data tests all pass with security on.